contents/teams/surveys/objectives.mdx

```diff
@@ -1,19 +1,37 @@
-### Objective: Surveys
+# Goal 1: Talk to customers
 
-Last quarter we polished up our new surveys UI.
-This quarter, we want to address some issues that came up, and then focus on building out some more ambitious features on top of Surveys.
+We'll only succeed if we are close to our customers, let's make our relationship with them stronger.
 
-Broadly, we will:
-1. Address outstanding experiments UI/UX issues
-2. Build no-code experiments
-3. Enterprise-3000 for surveys
-    * Global CSS
-    * Approvals
-    * Previews
-    * Progress bars for Surveys
-    * Back and Forward buttons for Multiple question surveys.
-    * Backend events for survey activations.
-    * Email surveys, which link out to full-screen surveys/ embed survey questions in them.
-    * Work on full-screen survey forms.
+- Do 5 customer interviews
+- Find the top 5 reference customers and build a relationship with them
+- Do a case study with a top reference customer
 
+# Goal 2: Improve quality
 
+There are a couple of reported bugs that were not addressed quickly, we'd like to speed up this process and fix them.
+
+- Reduce [SLA](https://en.wikipedia.org/wiki/Service-level_agreement) to fix bugs from months to weeks
+- Reduce [paper cut](https://en.wikipedia.org/wiki/Lingchi) issues by 50%
+    - Currently around ~30 [open bugs](https://github.com/PostHog/posthog/issues?q=is%3Aopen+is%3Aissue+label%3Afeature%2Fsurveys+label%3Abug)
+
+# Goal 3: Finish up [WIP](https://en.wikipedia.org/wiki/Work_in_process) features
+
+We've started and nearly finished a couple of features our customers requested, but they are not [GA](https://en.wikipedia.org/wiki/Software_release_life_cycle#General_availability_(GA)) yet, the goal is to finish them up, gather feedback and put them in from of customers.
+
+- [Adaptive Sampling](https://github.com/PostHog/posthog/issues/25722)
+    - Currently under a feature flag
+- [Partial Responses](https://github.com/PostHog/posthog/issues/19300)
+    - There are Draft PRs
+- [Survey Actions](https://github.com/PostHog/posthog/pull/23613)
+    - Currently under a feature flag
+- [AI Summaries](https://github.com/PostHog/posthog/pull/26046)
+    - Currently under early access features
+
```

Comment:
Pending https://github.com/PostHog/posthog/issues/27309 here, not sure if we should work on that yet.